package.json

```diff
@@ -0,0 +1,131 @@
+    "type": "git",
+    "url": "git+https://github.com/Hipo/hipo-frontend-ts-boilerplate.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Hipo/hipo-frontend-ts-boilerplate/issues"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.5.5",
+    "@babel/core": "7.5.5",
+    "@babel/plugin-proposal-class-properties": "7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "7.5.5",
+    "@babel/preset-env": "7.5.5",
+    "@babel/preset-react": "7.0.0",
+    "@babel/preset-typescript": "7.3.3",
+    "@hipo/eslint-config-base": "2.1.1",
+    "@hipo/eslint-config-react": "1.0.2",
+    "@hipo/stylelint-config-base": "1.0.0",
+    "@svgr/webpack": "4.3.2",
+    "@types/classnames": "2.2.9",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.5",
+    "@types/jest": "24.0.18",
+    "@types/pluralize": "0.0.29",
+    "@types/reach__router": "1.2.4",
+    "@types/react": "16.9.2",
+    "@types/react-dom": "16.9.0",
+    "@types/react-modal": "3.8.2",
+    "@types/react-redux": "7.1.2",
+    "@types/redux": "3.6.0",
+    "@typescript-eslint/eslint-plugin": "2.0.0",
+    "@typescript-eslint/parser": "2.0.0",
+    "autoprefixer": "9.6.1",
+    "babel-loader": "8.0.6",
+    "clean-webpack-plugin": "3.0.0",
+    "conventional-changelog-cli": "2.0.23",
+    "css-loader": "3.2.0",
+    "enzyme": "3.10.0",
+    "enzyme-adapter-react-16": "1.14.0",
+    "enzyme-to-json": "3.4.0",
+    "eslint": "6.2.1",
+    "eslint-config-prettier": "6.1.0",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-react": "7.14.3",
+    "eslint-plugin-react-hooks": "1.7.0",
+    "file-loader": "4.2.0",
+    "fork-ts-checker-webpack-plugin": "1.5.0",
+    "hipo-web-conventional-changelog": "github:Hipo/hipo-web-conventional-changelog",
+    "html-webpack-plugin": "3.2.0",
+    "husky": "3.0.4",
+    "jest": "24.9.0",
+    "lint-staged": "9.2.3",
+    "mini-css-extract-plugin": "0.8.0",
+    "node-sass": "4.12.0",
+    "postcss-loader": "3.0.0",
+    "postcss-normalize": "8.0.1",
+    "prettier": "1.18.2",
+    "sass-loader": "7.3.1",
+    "style-loader": "1.0.0",
+    "stylelint": "10.1.0",
+    "stylelint-no-unsupported-browser-features": "3.0.2",
+    "stylelint-order": "3.0.1",
+    "stylelint-scss": "3.9.4",
+    "terser-webpack-plugin": "1.4.1",
+    "ts-jest": "24.0.2",
+    "typescript": "3.5.3",
+    "url-loader": "2.1.0",
+    "webpack": "4.39.2",
+    "webpack-cli": "3.3.7",
+    "webpack-dev-server": "3.8.0",
+    "webpack-merge": "4.2.1",
+    "webpack-s3-plugin": "1.0.3"
+  },
+  "dependencies": {
+    "@hot-loader/react-dom": "16.9.0",
+    "@reach/router": "1.2.1",
+    "@sentry/browser": "5.6.2",
+    "axios": "0.19.0",
+    "classnames": "2.2.6",
+    "core-js": "3.2.1",
+    "pluralize": "8.0.0",
+    "query-string": "6.8.2",
+    "react": "16.9.0",
+    "react-dom": "16.9.0",
+    "react-hot-loader": "4.12.11",
+    "react-modal": "3.9.1",
+    "react-redux": "7.1.0",
+    "redux": "4.0.4",
+    "redux-saga": "1.0.5",
```

Comment:
Could we use tilda for these like:
`"redux-saga": "~1.0.5"`
That way we would get patch-level updates.

Reply:
We prefer to have exact control over versions for other packages